README-QUIC.md

```diff
@@ -7,12 +7,14 @@ client.
 Users interested in using the new QUIC functionality are encouraged to look at
 some of the following resources:
 
-- The [openssl-quic(7) manual page], which provides a basic reference overview
-  of QUIC functionality and how use of QUIC differs from use of TLS with regard
-  to our API;
 - The new [OpenSSL Guide], which provides introductory guides
   on the use of TLS, QUIC, and other OpenSSL functionality. See the
-  [ossl-guide-introduction(7) manual page][OpenSSL Guide] for the index.
+  [ossl-guide-introduction(7) manual page] for the index.
```

Comment:
`[OpenSSL Guide]` is an existing link but `[ossl-guide-introduction(7) manual page]` does not exist? But they would be supposedly the same links?